win32/ioutil.h

```diff
@@ -418,47 +416,42 @@ __forceinline static FILE *php_win32_ioutil_fopen(const char *patha, const char
 	return ret;
 }/*}}}*/
 
-__forceinline static int php_win32_ioutil_rename(const char *oldnamea, const char *newnamea)
+__forceinline static zend_result php_win32_ioutil_rename(const char *old_name_a, size_t old_name_a_len, const char *new_name_a, size_t new_name_a_len)
 {/*{{{*/
 	wchar_t *oldnamew;
 	wchar_t *newnamew;
-	int ret;
-	DWORD err = 0;
 
-	oldnamew = php_win32_ioutil_any_to_w(oldnamea);
+	oldnamew = php_win32_ioutil_conv_any_to_w(old_name_a, old_name_a_len, PHP_WIN32_CP_IGNORE_LEN_P);
```

Comment:
@cmb69 as don't have a Windows machine and can't test it, is such a function call change even sensible or not?
And if yes, can I also provide a `size_t` pointer for the 3rd argument so that `PHP_WIN32_IOUTIL_CHECK_PATH_W()` doesn't need to make a call to `wstrlen()`?

As seemingly this changes the behaviour of some tests in ways that I don't totally understand.